### PR TITLE
TP2000-617 Implement the new content for the TAP/TaMaTo homepage

### PR DESCRIPTION
**Pull request by dalecannon**
# TP2000-617 Implement the new content for the TAP/TaMaTo homepage

## Why
The homepage needs to have additional content added to it so that users can better understand the page that they have accessed. There is also a need to correct some accessibility inconsistencies on the page.

## What
Reduces font size of form label, removing multiple uses of H1 on page
Adds collapsible details component explaining what a workbasket is

## Checklist
- Requires migrations? No
- Requires dependency updates? No

Screenshot 2022-11-23 at 17 17 31

### CODE REVIEW

**Comment by gabelton** [comment]
Also Jess raised a point in the show and tell yesterday about the "This service lets you manage UK trade tariffs" and that this was too specific. I think she's right. There's a difference between talking about tariffs plural as a shorthand for duties and the UK tariff as the collection of all different components (measures, quotas, commodities etc.). I would change the line to "This service lets you manage the UK tariff"

File: common/forms.py [hunk 1]
```diff
@@ -172,9 +174,17 @@ class WorkbasketActions(TextChoices):
...
         super().__init__(*args, **kwargs)
         self.helper = FormHelper(self)
-        self.helper.legend_size = Size.EXTRA_LARGE
         self.helper.layout = Layout(
-            "workbasket_action",
+            Fieldset(
+                HTML.details(
+                    "What is a workbasket?",
+                    "A workbasket is used to collect all the elements that make up a tariff. "
+                    "Workbaskets will collect all the tariff changes you make for a specific tariff",
```
[the remark this answers is not in the document]
Comment (MPWilson92) [reply common/forms.py:182]: @dalecannon New wording after chatting to Gabe:

"A workbasket is used to collect all the changes you make to the UK's Import and Export Tariff data.

Workbaskets group these changes so they can be checked by Customs Declaration Services (CDS) before going live."

File: common/forms.py [hunk 2]
```diff
@@ -172,9 +174,17 @@ class WorkbasketActions(TextChoices):
...
         super().__init__(*args, **kwargs)
         self.helper = FormHelper(self)
-        self.helper.legend_size = Size.EXTRA_LARGE
         self.helper.layout = Layout(
-            "workbasket_action",
+            Fieldset(
+                HTML.details(
+                    "What is a workbasket?",
+                    "A workbasket is used to collect all the changes you make to the UK's Import and Export Tariff data. "
+                    "Workbaskets group these changes so they can be checked by Customs Declaration Services (CDS) before going live.",
```
Comment (gabelton) [inline common/forms.py:182]: --> Service

File: common/forms.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dalecannon) [reply common/forms.py]: I've updated it with the new wording.

**Review by gabelton — APPROVED** [review]

**Comment by gabelton** [comment]
You have a CI test failing. It's unrelated to your change. It fails intermittently. (Doug is working on a fix). Update your branch to rerun it and it hopefully will pass second time of asking

**Pull request merged**